.storybook/storybook.css

```
@@ -166,36 +166,36 @@
 /* Blockquote styling */
 .sbdocs-content blockquote {
-  border-left: 4px solid #0066cc;
+  border-left: 4px solid var(--pf-t--global--color--brand--default);
   padding-left: 1rem;
   margin: 1.5rem 0;
-  color: #666;
+  color: var(--pf-t--global--text--color--subtle);
   font-style: italic;
-  background: #f8f9fa;
+  background: var(--pf-t--global--background--color--secondary--default);
   padding: 1rem;
```

Stylelint (17.11.0)

[error] Overridden property "padding-left" by shorthand "padding" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)